skyvern/forge/sdk/routes/credentials.py

```diff
@@ -478,3 +481,107 @@ async def update_onepassword_token(
             status_code=500,
             detail=f"Failed to create or update OnePassword service account token: {str(e)}",
         )
+
+
+@base_router.get(
+    "/credentials/azure_client_secret_credential/get",
```

Comment:
```suggestion
    "/credentials/azure_credential/get",
```
Shall we do more concise name?